android/java/org/chromium/chrome/browser/settings/BraveMainPreferencesBase.java

```diff
@@ -377,6 +385,23 @@ private void updatePreferenceIcon(String preferenceString, int drawable) {
         }
     }
 
+    private void updateCustomAppIcon(String preferenceString) {
```

Comment:
I see lots of robolectric tests for fragments in upstream, are not currently not running those in Brave? If not, we definitely need to prioritize it and start adding them

Reply:
roboelectric it's not UI tests it's logic tests. We have examples of UI tests https://github.com/brave/brave-core/blob/master/android/javatests/org/chromium/chrome/browser/sync/BraveManageSyncSettingsTest.java

Reply:
@SergeyZhukovsky but this is a fragment, right? I see a lot of fragments in upstream that have robolectric tests? That being said, unit tests would also be good here.